releases/v2.0.1.md

```diff
@@ -0,0 +1,106 @@
+---
+title: What&#39;s New in v2.0.1
+toc: false
+summary: Additions and changes in CockroachDB version v2.0.1 since version v2.0.0
+---
+
+## April 23, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0.1.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0.1.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0.1.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0.1.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### General Changes
+
+- Added a [cluster setting](../v2.0/cluster-settings.html) for HLC to be monotonic across restarts. [#24624][#24624]
```

Comment:
I think it would be more informative to also provide the name of the new cluster setting here (ditto for the item below). It would also make this release note more symmetric with the other notes about new settings in the "enterprise changes" section.

Reply:
Done.